### DISCUSSION TITLE
New Halmet have issue with USB device not recognized

### DISCUSSION THREAD

**Discussion by Steve7848** [hatlabs/discussions#106] [Q&A]
I just got a new Halmet and the issue I am having is I get a windows error USB device not recognized.

I installed Visual Studio and PlatformIO. I downloaded the zip folder with the Zip folder example code and flashed the example onto the board. Now the computer keeps acting like I unplug and plug in a USB device with the chime. If I use Visual Studio to look at the COM ports I see COM7 keeps disappearing and appearing so I can't use terminal monitor to see what is going on.  The error Terminal monitor gives is 
Disconnected (ClearCommError failed (PermissionError(13, 'Access is denied.', None, 5)))

Any recommendations? Both PlatformIO and Visual Studio Code are the latest versions as they were freshly installed on Windows 10.

**Comment by mairas** [comment]
My first suggest would be to try another USB cable. It's surprising how often they are the culprit to any problems.

**Reply by Steve7848** [reply]
It seems to work if I set the WiFi SSID and password under main.cpp line 175.
